airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -810,22 +826,19 @@ def execute(self, context):
                 cluster_fields=self.cluster_fields,
                 labels=self.labels,
                 view=self.view,
-                encryption_configuration=self.encryption_configuration
+                encryption_configuration=self.encryption_configuration,
+                table_resource=self.table_resource,
             )
-            self.log.info('Table created successfully: %s:%s.%s',
-                          self.project_id, self.dataset_id, self.table_id)
-        except HttpError as err:
-            if err.resp.status != 409:
-                raise
-            else:
-                self.log.info('Table %s:%s.%s already exists.', self.project_id,
-                              self.dataset_id, self.table_id)
+            self.log.info('Table %s.%s.%s created successfully',
+                          table.project, table.dataset_id, table.table_id)
+        except Conflict:
```

Comment:
Will it be able to catch `Conflict` exception here without specifying `exists_ok=False` when calling `create_empty_table` method?

Reply:
Good catch! Will fix